STARTERKIT/js/script.js

```diff
@@ -1,31 +0,0 @@
-/**
```

Comment:
Why we need to remove script.js file?
This file will be included in libraries definition
https://github.com/skilld-labs/zen/blob/8.x-7.x/STARTERKIT/STARTERKIT.libraries.yml#L15